### DISCUSSION TITLE
Resize desktop resolution

### DISCUSSION THREAD

**Discussion by victorlissoviy** [Xpra-org/xpra#3804] [Q&A] [answered]
Hello, I have a clean docker container ubuntu:22.04 with systemd. I installed xpra and icewm graphical environment on it.
I have a problem: when changing the dimensions of the browser window, the extension in the graphical environment xpra and icewm does not change. The connection is made through the Google Chrome browser 111.0.5563.64 and Firefox 110.0.1. The situation is detected only during the "start-desktop" connection, during the "connect" connection everything works well.
Please tell me how to fix the situation.

My docker file
[Dockerfile.txt](https://github.com/Xpra-org/xpra/files/11052186/Dockerfile.txt)

Also the settings I use for xpra and xpra-html5
[xpra.conf.txt](https://github.com/Xpra-org/xpra/files/11052191/xpra.conf.txt)
[default-settings.txt](https://github.com/Xpra-org/xpra/files/11052193/default-settings.txt)

I run the docker container with the following command
```shell
docker run -d --privileged \
    --volume=/sys/fs/cgroup:/sys/fs/cgroup:rw \
    --cgroupns=host --name ubuntu-xpra ubuntu-xpra-sysd:22.04
```

I use Linux OpenSuse Tumbleweed
Linux version 6.2.6-1-default (geeko@buildhost) (gcc (SUSE Linux) 12.2.1 20230124 [revision 193f7e62815b4089dfaed4c2bd34fd4f10209e27], GNU ld (GNU Binutils; openSUSE Tumbleweed) 2.40.20230127-2) #1 SMP PREEMPT_DYNAMIC Mon Mar 13 10:57:27 UTC 2023 (fa1a4c6)

Thank You.
If need some more information, tell me please.

**Comment by totaam** [comment]
> The situation is detected only during the "start-desktop" connection, during the "connect" connection everything works well.

I don't understand what that means.
Please specify which versions you have installed.

**Reply by victorlissoviy** [reply]
I connect to the xpra server through an internet browser (xpra-html5).
When I connect to the xpra server using the "start-desktop" connection, then running the graphical environment icewm and this environment does not change its size after I changed the size of the browser window. But when I use the "connect" connection, then running only one program xterm. I maximize the window of this application, then I change the size of the browser window from which I logged on to the server and then the size of the xterm application window changes.

**Reply by victorlissoviy** [reply]
xpra version 4.4.4
xpra showconfig [config.txt](https://github.com/Xpra-org/xpra/files/11052540/config.txt)
xpra info [info.txt](https://github.com/Xpra-org/xpra/files/11052541/info.txt)

**Reply by victorlissoviy** [reply]
Hello, do you need more information about this question?

**Reply by totaam** [reply]
No, just time to look into it.

**Comment by totaam** [comment, marked as answer]
Please try with a better distro - Debian and Ubuntu are plagued by display resolutions and DPI issues:
https://github.com/Xpra-org/xpra/issues/3989#issuecomment-1720383100
